components/patient-data/indexing/src/main/java/org/phenotips/data/indexing/internal/SolrPatientIndexer.java

```diff
@@ -92,9 +101,25 @@ public void index(Patient patient)
         }
         input.setField("reporter", reporter);
 
+        // Index direct phenotypes and extended ancestor sets
         for (Feature phenotype : patient.getFeatures()) {
-            input.addField((phenotype.isPresent() ? "" : "negative_") + phenotype.getType(), phenotype.getId());
+            String presence = (phenotype.isPresent() ? "" : "negative_");
+            String fieldName = presence + phenotype.getType();
+            String ancestorFieldName = "extended_" + presence + "phenotype";
```

Comment:
The goal was to have a combined set of all extended phenotypes for patient similarity searching. If you'd like to have all the `extended_*phenotype` fields as well for some other purpose, then we can instead create an additional field that is the combination of all of those.